.gitignore

```diff
@@ -1 +1,2 @@
+.idea/
```

Comment:
Why this?

Reply:
It's my IDE local settings in there, I can use a global gitignore if you don't want this here ... just tell me

Reply:
I prefer that you delete it. Thank you.

Reply:
done